docroot/modules/custom/foia_webform/src/Plugin/WebformHandler/FoiaEmailWebformHandler.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace Drupal\foia_webform\Plugin\WebformHandler;
+
+use Drupal\file\Entity\File;
+use Drupal\webform\Plugin\WebformHandler\EmailWebformHandler;
+use Drupal\webform\WebformSubmissionInterface;
+use Drupal\node\Entity\Node;
+
+/**
+ * Emails a webform submission.
+ *
+ * @Webformhandler(
+ *   id = "foia_email",
+ *   label = @Translation("FOIA email"),
+ *   category = @Translation("Notification"),
+ *   description = @Translation("Sends a webform submission to appropriate the agency."),
+ *   cardinality = \Drupal\webform\WebformHandlerInterface::CARDINALITY_UNLIMITED,
+ *   results = \Drupal\webform\WebformHandlerInterface::RESULTS_PROCESSED,
+ * )
+ */
+class FoiaEmailWebformHandler extends EmailWebformHandler {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function sendMessage(WebformSubmissionInterface $webform_submission, array $message) {
+    // Get form submissions.
+    $data = $webform_submission->getData();
+    $error_message = NULL;
+    $context = [];
+
+    // If there is a file attachment, get the file URL.
+    if (isset($data['attachments_supporting_documentation'])) {
+      $file = File::load($data['attachments_supporting_documentation']);
+      $file_url = file_create_url($file->getFileUri());
+      $data['attachments_supporting_documentation'] = $file_url;
+    }
+
+    // Format the submission values as CSV.
+    $submissions = $this->arrayToString($data);
```

Comment:
Giving this variable a plural structure (submissions vs submission) makes it sound like we're sending multiple submissions. Was there a reason you gave it that name?

Reply:
Good point. I will change it to `$csv_string` for clarity.